pkgs/hwi/default.nix

```diff
@@ -1,11 +1,18 @@
 { stdenv, fetchurl, fetchFromGitHub, python3 }:
 
 with stdenv.lib;
-with python3.pkgs;
 
 let
-  buildInputs = [ mnemonic ecdsa typing-extensions hidapi libusb1 pyaes ];
+  python = python3.override {
+    packageOverrides = self: super: {
+      # HWI requires mnemonic <0.19 but nixpkgs has a newer version
+      mnemonic = self.callPackage ./mnemonic {};
```

Comment:
Adding our own packages seems too complicated at first. One could override only the version and sha256. But own packages are probably better in case something else changes about the mnemonic and ecdsa nixpkgs other than just those two attributes.